gedi-subset/src/gedi_subset/gedi_utils.py

```diff
@@ -344,9 +345,20 @@ def subset_hdf5(
     `"BEAM*"` groups.
     """
 
+    def datasets(group: h5py.Group) -> Iterable[h5py.Dataset]:
+        """Returns a flattened h5py.Group of one dimensional values"""
```

Comment:
In Python docstrings, the general convention is to use active voice, rather than passive voice, at least for the first sentence. For example, "Return ..." rather than "Returns ...". See https://peps.python.org/pep-0257/

```suggestion
        """Return iterable of all 1-dimensional ``h5py.Dataset``s at all levels within
        an ``h5py.Group``.
        """
```

In conjunction with this, let's tweak the docstring for the `subset_hdf5` function by removing the term `top-level`, since we now traverse all depths.